command/harness/destroy.go

```diff
@@ -76,6 +80,20 @@ func handleDestroy(ctx context.Context, r *VMCleanupRequest, s store.StageOwnerS
 		WithField("instance_id", inst.ID).
 		WithField("instance_name", inst.Name)
 
+	logr.Traceln("invoking lite engine cleanup")
+	client, err := lehelper.GetClient(inst, env.Runner.Name, inst.Port, env.LiteEngine.EnableMock, env.LiteEngine.MockStepTimeoutSecs)
+	if err != nil {
+		logr.WithError(err).Errorln("could not create lite engine client for invoking cleanup")
```

Comment:
if err happens here, it will cause null pointer error at L88 since client is null